apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -102,6 +102,14 @@ type AdvancedConfig struct {
 	HorizontalPodAutoscalerConfig *HorizontalPodAutoscalerConfig `json:"horizontalPodAutoscalerConfig,omitempty"`
 	// +optional
 	RestoreToOriginalReplicaCount bool `json:"restoreToOriginalReplicaCount,omitempty"`
+	// +optional
+	ComplexScalingLogic ComplexScalingLogic `json:"complexScalingLogic,omitempty"`
+}
+
+// ComplexScalingLogic describes advanced scaling logic options like formula
+type ComplexScalingLogic struct {
```

Comment:
other possible name suggestion from Jorge was `modifiers`

Reply:
I think that modifiers could be mor user-friendly as we plan to include the formula but also the external modifier in the near future.
@zroubalik  @tomkerkhove ?